src/Modifier/Product/VariantAttributesModifier.php

```diff
@@ -54,7 +58,17 @@ class VariantAttributesModifier extends Modifier
 
     private Connection $database;
 
-    private string $activeSnippet;
+    /**
+     * @var BaseModel
+     */
+    private $model = null;
```

Comment:
Type hint is missing.

```suggestion
    private ?BaseModel $model = null;
```